espresso/device/java/androidx/test/espresso/device/util/UiAutomationUtil.kt

```diff
@@ -0,0 +1,15 @@
+package androidx.test.espresso.device.util
+
+import android.app.Instrumentation
+import android.os.ParcelFileDescriptor.AutoCloseInputStream
+import java.nio.charset.Charset
+
+/** Collection of utility methods using UiAutomation APIs */
+fun executeShellCommand(instrumentation: Instrumentation, command: String): String {
```

Comment:
Please make it an extension function for Kotlin and it'll still have this signature for Java. Maybe add a `@file:JvmName` as My endi